src/garvis/heartbeat_kernel.py

```
@@ -0,0 +1,449 @@
+                    plan=plan_payload,
+                    protected_action=False,
+                    artifact_sha256=artifact_sha,
+                    status=CycleStatus.COMPLETED,
+                    raw_post=raw_post,
+                    diff=observed_diff,
+                    verification=verification,
+                    contradictions=contradictions,
+                    self_claims=claims,
+                    provenance=dict(provenance or {}),
+                )
+            except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)